cms/static/sass/elements/_modules.scss

```diff
@@ -131,6 +131,7 @@
         height: ($baseline*5);
         margin-bottom: ($baseline/2);
         border: 1px solid $green-d2;
+        padding: 0;
```

Comment:
I thought padding was considered inside of a border. Should this move one line down, below border-radius? And the box-shadow would be above margin-bottom?